### ISSUE TITLE
Fix display of covreach graph

### ISSUE THREAD

**Issue by fredher** [ticktac-project/tchecker#31]
Current graph display traverses the graph using a DFS. However, the graph may not be connected. Hence, display would be incomplete. We should instead traverse the list of nodes, display each node, and all its outgoing edges

**Comment by pictavien** [comment]
Hi Fred,

Do you have a test-case to check the results ? 

Thanks.

**Comment by fredher** [comment]
Hi,

I think there is one of the current examples that exhibit the problem when using extrapolation ExtraLU+l, but I don't remember which one :-). I will design an example, but it may take me some time. Are you working on this issue? I need to rewrite the covreach algorithm (see #33 ), and the associated graph. I had planned to fix this issue at the same time.

Best regards,
Frédéric

**Comment by fredher** [comment]
I am totally confident that you did it well :-)

Regarding the test case, could you run the tests with your fix and see if some existing NR test fails? This would mean that you have identified the example :-). Otherwise, I will design an example. That is not easy because, this is particular to zone graphs for which zone inclusion is not a simulation relation (this is the case for extrapolation ExtraLU+).

Best regards,
Frédéric

**Comment by pictavien** [comment]
We don't have, for the time being, a test suite for dot output of covreach algorithm because nodes
are identified by their addresses. I have done a fix also for this issue but I'm sure it follows the API 
style. I commit all that in few minutes ...

**Comment by fredher** [comment]
True, I forgot this issue. I should add a node identifier that does not depend on the address. Plus we need to order the nodes to ensure deterministic output. This is an issue already raised by Alexandre (see #23 )
